Library/Formula/emp.rb

```diff
@@ -0,0 +1,27 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/emp"
+  url "https://github.com/remind101/emp/archive/v0.9.1.tar.gz"
+  version "0.9.1"
+  sha256 "9619ffd25856988d92d39b31d9e7ed6edacbd85543ff6e60300954df7c16135d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    emppath = buildpath/"src/github.com/remind101/emp"
+    emppath.install Dir["{*,.git}"]
+
+    cd emppath do
+      system "go", "build", "-o", bin/"emp"
+    end
+  end
+
+  test do
+    assert_equal "#{version}",
```

Comment:
If I run the tests twice, it fails because the port is still bound. 

```
~ brew test emp
Testing emp
31824
==> go run /private/tmp/emp20151013-31812-17nf07w/test-server.go
==> EMPIRE_API_URL=http://127.0.0.1:8035 /usr/local/Cellar/emp/0.9.1/bin/emp releases -a foo
~ lsof | grep 8035
test-serv 31829 stevenberlanga    3u    IPv6 0xc5f2310168a54eef         0t0      TCP *:8035 (LISTEN)
~ ps aux | grep 31829
stevenberlanga  31871   0.0  0.0  2434836    768 s001  R+   10:00AM   0:00.00 grep 31829
stevenberlanga  31829   0.0  0.0 573394348   3548 s001  S     9:58AM   0:00.01 /var/folders/vp/d2lk3_ms5dvb4y69wdy4z5800000gn/T/go-build411514785/command-line-arguments/_obj/exe/test-server
```

I tried using [Process.spawn](http://ruby-doc.org/core-2.2.0/Process.html#method-c-spawn) instead but still ran into the same issue